src/chainconsumer/chain.py

```diff
@@ -281,6 +282,12 @@ def _validate_model(self) -> Chain:
         if self.num_free_params is not None:
             assert np.isfinite(self.num_free_params), "num_free_params is not finite"
 
+        if self.multimodal and self.statistics is not SummaryStatistic.HDI:
```

Comment:
I realise that the other way of doing this would be to change the default for `statistics` to be `None`, and then if its None (aka unspecified) then you just set it to HDI, otherwise (if not multimodal) set it to Max. This would mean the user only hits this ValueError if they deliberately try and set the wrong method instead of just forgetting to set it